### ISSUE TITLE
Upload select and print options not working

### ISSUE THREAD

**Issue by jefflongo** [dougbrion/OctoRest#29]
This works:
```python
hets.upload("test.gcode")
hets.select("test.gcode", print=True)
```

This does not:
```python
hets.upload("test.gcode", select=True, print=True)
```

The file is uploaded, but not selected (and does not start printing, obviously). 

I suspect it has to do with how the arguments are formatted. Here is a function that works properly that I wrote before finding this library:

```python
def upload_and_run(path: str | os.PathLike) -> None:
    url: str = f"{URL}/api/files/local"
    headers: dict[str, str] = {
        "X-Api-Key": API_KEY,
    }

    with open(path, "rb") as file:
        files: dict[str, tuple] = {
            "file": (os.path.basename(path), file),
        }
        data: dict[str] = {
            "select": True,
            "print": True,
        }
        response = requests.post(url, headers=headers, files=files, data=data)
        if response.status_code != 201:
            error: str = json.loads(response.content)["error"]
            raise IOError(f"Failed to upload: {error}")
```

**Comment by dougbrion** [comment]
Thanks for catching this! Do you want to submit a pull request and we can merge this into the library.

**Comment by jefflongo** [comment]
Sure I'll take a look at this in more detail soon.

**Comment by jefflongo** [comment]
After further testing, this issue has been fixed at some point between v0.4 and master. My apologies. I had only installed the library via PyPI.
